reasoner/resolution/resolver/ConcludableResolver.java

```diff
@@ -69,6 +75,7 @@ public ConcludableResolver(Driver<ConcludableResolver> driver, Concludable concl
         this.applicableRules = new LinkedHashMap<>();
         this.recursionStates = new HashMap<>();
         this.requestStates = new HashMap<>();
+        this.missingBounds = missingBounds(concludable.pattern());
```

Comment:
I'm also thinking that this could be better named `unboundVars`, as they aren't _missing_ per se